RptToXml/RptDefinitionWriter.cs

```diff
@@ -74,20 +76,29 @@ public void WriteToXml(XmlWriter writer)
 			writer.Flush();
 		}
 
+		private static readonly System.Text.RegularExpressions.Regex CompiledRegexp =
+			new System.Text.RegularExpressions.Regex("[^\\u0009\\u000a\\u000d\\u0020-\\uD7FF\\uE000-\\uFFFD]",
+				System.Text.RegularExpressions.RegexOptions.Compiled);
+
+		private static void WriteAttributeString(XmlWriter writer, string name, string value)
+		{
+			string myValue = value == null ? null : CompiledRegexp.Replace(value, "");

```

Comment:
How much slower is it to run the regex replace across the board? I wonder if people who are processing dozens of reports will suffer here. Should we make this a commandline option?

Reply:
I tested on 1100 reports and the time only increased by 25 seconds (total)... 24:24 min > 24:49 mins. I think the processing time is not a big enough issue to make it a command line option. 

I did run into an issue w/ another character however, I put a comment blow.